src/utilities/mounted-ref.ts

```diff
@@ -0,0 +1,13 @@
+import {useRef, useEffect} from 'react';
+
+export function useMountedRef() {
+  const mounted = useRef(true);
```

Comment:
Should this be false? 🤔 
```suggestion
  const mounted = useRef(false);
```